src/sentry/api/endpoints/organization_group_index.py

```diff
@@ -183,7 +196,33 @@ def get(self, request, organization):
 
         results = list(cursor_result)
 
-        context = serialize(results, request.user, serializer())
+        snuba_filters = []
+        if query_kwargs["search_filters"] is not None:
+            snuba_filters = [
+                convert_search_filter_to_snuba_query(search_filter)
+                for search_filter in query_kwargs["search_filters"]
+                if search_filter.key.name not in self.skip_snuba_fields
+            ]
```

Comment:
Minor nitpick, this block from 199-205 could move down to just before line 211. It will define the variables closer to their use and read a bit nicer. Another side benefit is that most customers to start won't be getting dynamic counts and thus won't be doing this work.

Reply:
If we change the way we generate context, ie. as a subset of lifetime stats, this portion of the code will change as well so I won't move it for now